.grit/patterns/python/use_flask_jsonify.md

```diff
@@ -0,0 +1,87 @@
+---
+title: use `flask.jsonify()`
+tags: [fix, best-pactice, flask]
+---
+
+`flask.jsonify()` is a `Flask` helper method which handles the correct settings for returning `JSON` from `Flask` routes
+
+### references
+
+- [flask.json.jsonify](https://flask.palletsprojects.com/en/2.2.x/api/#flask.json.jsonify)
+
+```grit
+engine marzano(0.1)
+language python
+
+`def $func($params):$funcBody` where {
```

Comment:
We should filter this to only routes. Look for a decorator.

Reply:
updated it now